v1/accountapplications/accountapplications.gunk

```diff
@@ -0,0 +1,285 @@
+//         },
+//         Consumes: []string{
+//                 "application/json",
+//         },
+//         Produces: []string{
+//                 "application/json",
+//         },
+//         Responses: map[string]openapiv2.Response{
+//                 "400": openapiv2.Response{
+//                         Description: "Returned when the request body is malformatted or does not match the expected request.",
+//                 },
+//                 "401": openapiv2.Response{
+//                         Description: "Returned when the request does not contains the user's credentials.",
+//                 },
+//                 "403": openapiv2.Response{
+//                         Description: "Returned when the user does not have permission to access the resource.",
+//                 },
+//                 "404": openapiv2.Response{
+//                         Description: "Returned when the resource is not found.",
+//                 },
+//                 "500": openapiv2.Response{
+//                         Description: "Returned when an unexpected error occured on the server side.",
+//                 },
+//         },
+//         SecurityDefinitions: openapiv2.SecurityDefinitions{
+//                 Security: map[string]openapiv2.SecurityScheme{
+//                         "OAuth2": openapiv2.SecurityScheme{
+//                                 Type:             openapiv2.TYPE_OAUTH2,
+//                                 Flow:             openapiv2.FLOW_ACCESS_CODE,
+//                                 Description:      "Access to transactions resources is restricted to authenticated users. The authentication process follows the Authorization Code flow.",
+//                                 AuthorizationURL: "/v1/auth",
+//                                 TokenURL:         "/v1/token",
+//                                 Scopes: openapiv2.Scopes{
+//                                         Scope: map[string]string{
+//                                                 "read":  "Grants read access",
+//                                                 "write": "Grants write access",
+//                                         },
+//                                 },
+//                         },
+//                 },
+//         },
+// }
+package accountapplications
+
+import (
+	"time"
+
+	"github.com/gunk/opt/http"
+	"github.com/gunk/opt/openapiv2"
+
+	types "github.com/openbank/openbank/v1/types"
+)
+
+// Entity type defines the type of account application
+type AccountApplicationStatus int
+
+const (
+	UnknownApplicationStatus AccountApplicationStatus = iota
+
+	// Account application is created
+	Requested
+
+	// Account application is accepted
+	Accepted
+)
+
+// Structure of user information
+type User struct {
+	UserID     string `pb:"1" json:"user_id"`
+	Email      string `pb:"2" json:"email"`
+	ProviderID string `pb:"3" json:"provider_id"`
+	Provider   string `pb:"4" json:"provider"`
```

Comment:
what is provider means?
maybe put a related comment here

Reply:
hm. From the open back project, the values returned for provider_id and provider from the example are stated below. I can't guess what it can be, maybe a bank connector. I',m not sure. so it's better to remove it from here, also the userid is probably the online credentials to login. Any guess here? @jwangsadinata @nkristianto 

```
    "provider_id":"Chris",
    "provider":"http://127.0.0.1:8080" 
```

Reply:
for me, we can just make it clear with some comments right now. Would be better if we can define the source as a const on types package. wdyt @jwangsadinata @ghazninattarshah

Reply:
I removed the provider fields

Reply:
@nkristianto What source do you want to define as const?